packages/core/nuxt-theme-module/theme/pages/Category.vue

```diff
@@ -440,6 +443,16 @@ export default {
       changeFilters(selectedFilters.value);
     };
 
+    const addToCart = async (product, quantity) => {
+      await addItemToCart({ product, quantity });
+      sendNotification.cart.addItem({
+        productName: product._name,
+        error: error.value.addItem ? error.value.addItem.message : false,
+        onClick: () => $router.push(`/checkout/${isAuthenticated.value ? 'shipping' : 'personal-details'}`),
```

Comment:
```suggestion
        onClick: () => $router.push(`/checkout/`)
```
 isn't that enough since we have `middleware/checkout.js` ?

Reply:
`middleware/checkout.js` does not handle `/checkout/` (we have server error), so I changed it to:

```js
 onClick: () => $router.push(`/checkout/personal-details`)
```